superset-frontend/src/explore/components/controls/DndColumnSelectControl/types.ts

```diff
@@ -16,19 +16,60 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { ColumnMeta } from '@superset-ui/chart-controls';
+import { ReactNode } from 'react';
+import { AdhocFilter } from '@superset-ui/core';
+import {
+  BaseControlConfig,
+  ColumnMeta,
+  Metric,
+} from '@superset-ui/chart-controls';
+import { DatasourcePanelDndItem } from '../../DatasourcePanel/types';
+
+export const GroupByItemType = 'groupByItem';
+export const FilterItemType = 'filterItemType';
```

Comment:
https://github.com/apache/superset/pull/13210/files#r583265905

I'd imagine you will be adding more types in the future, so it could make sense to make this an enum.